### PR TITLE
Create ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57827](https://cumulocity.atlassian.net/browse/MTM-57827)
Original PR: [5487](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5487)

### CODE REVIEW

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: crl c8y-help (#5440) [GRAFT][release/y2024] (#5487)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Help Component Update
```

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57827
+version: 1018.503.52
+---
+crl c8y-help (#5440) [GRAFT][release/y2024] (#5487)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:17]: The Claude AI suggested to use the following body:
```suggestion
The help component has been updated in this release. This update includes improvements to the search functionality, making it easier for users to find the information they need. A few minor bug fixes and optimizations have also been implemented to improve performance and stability of the help system.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by janhommes** [comment]
@mmed-softwareag please check this change note. Is this accurate? Maybe you also need to create one for develop, as this one is the one for yearly. Please update the files directly.

**Review by janhommes — APPROVED** [review]

**Comment by janhommes** [comment]
@mmed-softwareag can you also create a PR for develop for this?

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57827
+version: 1018.503.52
+---
+The information regarding certificate management along with a link to the documentation is available on the certificate settings page.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:17]:
```suggestion
Information about certificate management along with a link to the documentation has been added on the **Settings** tab of the **Trusted certificates** page.
```

Comment (BeateRixen) [reply content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:17]: Or does the information show up on the **Trusted certificates** page in general?

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Additional information about certificate settings page.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:3]:
```suggestion
title: Added information about certificate management
```

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+product_area: Platform services
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:4]: Should be "Device management app", shouldn´t it @janhommes ?

Comment (janhommes) [reply content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:4]: No, guess this is with A&A. Can you please ping the developer @mmed-softwareag? You can always get the contributor from the PR. I can only do the reviews for the AppEnablement team.

Comment (mmed-softwareag) [reply content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md]: correct, this is part of trusted certificates in the Device management app.

Comment (mmed-softwareag) [reply content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md]: changed.

**Review by mmed-softwareag — APPROVED** [review]

**Comment by janhommes** [comment]
@BeateRixen can we merge this?

**Comment by BeateRixen** [comment]
> @BeateRixen can we merge this?

@janhommes  @mmed-softwareag has not reacted on your comments so far. Moreover, the product area and the component do not match, see my comment.

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added information about certificate management
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: q3kclF6pO
+    label: Authentication
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:10]: Should probably be "Device Management app"

Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:10]:
```suggestion
    label: Device Management app
```

File: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added information about certificate management
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: q3kclF6pO
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md:9]:
```suggestion
  - value: component--KIsStyzM 
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@l2p- Please check if this is correct and has been added to the develop branch as well if needed.

**Pull request merged**